react-native-libraries.json

```diff
@@ -13013,5 +13013,17 @@
     "android": true,
     "web": true,
     "expoGo": true
-  }
+  },
+   {
+    "githubUrl": "https://github.com/AakashThakur23102000/useapihook-dexterverse",
+    "npmPkg": "useapihook-dexterverse",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "windows": true,
+    "macos": true,
+    "tvos": true,
```

Comment:
Hello @AakashThakur23102000, thank you for adding new entry to the directory! 👍 

For marking the out-of-tree platforms support we require an working example. If you can provide some, let's add them to `examples` array, otherwise let's remove those platform from the entry:
* https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms